test/include/test/jemalloc_test.h.in

```diff
@@ -132,6 +132,12 @@ static const bool config_debug =
 #define MEXP 19937
 #include "test/SFMT.h"
 
+#ifndef JEMALLOC_HAVE_MALLOC_SIZE
+#define TEST_MALLOC_SIZE malloc_usable_size
+#else
+#include <malloc/malloc.h>
```

Comment:
This is required, because `malloc_size` is declared in `<malloc/malloc.h>`. But `malloc_usable_size` is declared in `<malloc.h>`, why isn't it required to include that header?

Reply:
malloc related definitions are covered by <stdlib.h> (included at the beginning of this file), which is part of the C standard. https://man7.org/linux/man-pages/man0/stdlib.h.0p.html

Reply:
Yes, but I'm asking specifically about `malloc_usable_size`. It's not in `<stdlib.h>` and requires `<malloc.h>` to be included.

Reply:
Ah that's right. It should be from "jemalloc/jemalloc.h" (where we have name mangling: `#  define je_malloc_usable_size malloc_usable_size`, generated through https://github.com/jemalloc/jemalloc/blob/dev/include/jemalloc/jemalloc_rename.sh)

jemalloc.h is then is included in `test/jemalloc_test.h`.

But that should also mean the header for `malloc_size` isn't necessary. Let me verify that.

Reply:
@Lapenkov As suspected the header wasn't needed. Removed it and the builds were fine. Checked the log of the MacOS builds, it has `malloc_size` detected properly, and didn't have additional warnings.

Reply:
Great, this is what I was thinking. Logically, either two of them should be required, or none.